db/migrate/20170109161429_create_ownerships.rb

```diff
@@ -0,0 +1,8 @@
+class CreateOwnerships < ActiveRecord::Migration[5.0]
+  def change
+    create_table :ownerships do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
We won't need the ownerships table in this case; I think this was automatically generated when creating the model from the command line with 'rails generate...' 
To remove this table from the code we need to remove the migration, and also recreate the 'db/schema.rb' file without the table. You can do it like this :
1. Remove the migration from the repository:
- git rm db/migrate/20170109161429_create_ownerships.rb
2. Reset the schema file to the previous content in the branch named 'devel'
- git checkout devel -- db/schema.rb
3. Commit changes and push again to your github branch to update the pull request